README.md

```diff
@@ -4,24 +4,16 @@ Version 7.0.0 (beta)
 This is the the develop branch of the Csound main code repository. At
 the moment, we are developing the next main version (moving from 6.x
 to 7.0), and this branch reflects current work-in-progress. This is
-still undergoing changes and fixes until the first release. The
```

Comment:
Odd, looks like the README is somehow changed back to what it was prior to the other PR here?

Reply:
I think this happens because it is always rewritten by CMake. I needed to have merged before doing a build.